demos/spaceships/src/server.rs

```diff
@@ -184,6 +192,68 @@ const NAMES: [&str; 35] = [
     "Mr. T",
 ];
 
+#[derive(Resource)]
+struct ServerStallStress {
```

Comment:
Could you add a short comment explaining what this does?
Something like 
"Sleep the thread by `duration` every `interval_ticks` to see how the timeline sync logic handles timeline mismatches"